packages/contracts-bedrock/deploy-config/devnetL1.json

```diff
@@ -45,5 +45,6 @@
   "l1StartingBlockTag": "earliest",
   "l2GenesisRegolithTimeOffset": "0x0",
   "faultGameAbsolutePrestate": 140,
-  "faultGameMaxDepth": 4
-}
\ No newline at end of file
+  "faultGameMaxDepth": 4,
+  "faultGameDuration": 604800
```

Comment:
Maybe name this `faultGameMaxDuration`? A bit more descriptive of what the value actually means.